packages/generator/shared.js

```diff
@@ -85,6 +85,16 @@ function filepathForFontVariant(webfont, variantKey) {
   return path.join(FontAssetsDir, filenameForFontVariant(webfont, variantKey));
 }
 
+function getNextPackageVersion(packageDir, options) {
+  if (!options || !options.patch || !fs.existsSync(packageDir)) {
+    return PackageVersion;
+  }
+
+  const packageJson = require(path.join(packageDir, 'package.json'));
+  const [major, minor, patch] = packageJson.version.split('.');
+  return `${major}.${minor}.${Number(patch) + 1}`;
+}
```

Comment:
We could use `semver` package for easier version management. (only suggestion, feel free to ignore)